### DISCUSSION TITLE
Nextcloud sync just stopped

### DISCUSSION THREAD

**Discussion by hanserasmus** [photoprism/photoprism#1758] [General]
Hi everyone

I have successfully set up Photoprism to sync with Nextcloud, but on the 13th of November, whilst on holiday it just stopped. Under Library>Erros I just get "unexpected EOF" messages. Nextcloud is running properly, the folder is still correct. I don't mind Nextcloud folder to photoprism. The latter needs to download the photos. Any ideas how I can get this going again? Is it stuck on something that needs manual intervention? I have about 16GB of swap configured now, not sure if that may have been an issue at some point.
Thank you in advance for any help!
Cheers
Hans

**Comment by graciousgrey** [comment]
Seems to be the same issue as: https://github.com/photoprism/photoprism/issues/1781

**Comment by hanserasmus** [comment]
Thank you @graciousgrey. Yes, indeed it does. I will look into some workarounds mentioned in the comments and look into it. Thank you!
